crates/orchestrator/src/telemetry.rs

```diff
@@ -161,7 +156,10 @@ mod tests {
         env::set_var("TRACING_LEVEL", "info");
```

Comment:
do we need to remove `TRACING_LEVEL` from everywhere in code?

Reply:
Fixed.